modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -238,6 +238,13 @@ if (CONFIG_BUILD_WITH_TFM)
       )
   endif()
 
+  if (CONFIG_FPU AND CONFIG_FP_HARDABI)
+    list(APPEND TFM_CMAKE_ARGS -DCONFIG_TFM_FP=hard)
+    list(APPEND TFM_CMAKE_ARGS -DCONFIG_TFM_FP_ARCH=${FPU_FOR_${GCC_M_CPU}})
+  else()
+    list(APPEND TFM_CMAKE_ARGS -DCONFIG_TFM_SPE_FP=soft)
```

Comment:
Why does this use `CONFIG_TFM_SPE_FP` and not `CONFIG_TFM_FP`? Also looking at the commit history, `CONFIG_TFM_SPE_FP` seems to be no longer a thing?

Reply:
Thanks, this is a mistake.